nixos/modules/services/networking/iperf.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }: with lib;
+let
+  cfg = config.services.iperf3;
+
+  api = {
+    enable = mkEnableOption "iperf3 network throughput testing server";
+    port = mkOption {
+      type        = types.ints.u16;
+      default     = 5201;
+      description = "Server port to listen on for iperf3 client requsts.";
+    };
+    affinity = mkOption {
+      type        = types.nullOr types.str;
```

Comment:
Ah, so yeah the `n/m` syntax doesn't make sense for the server, so only the `n` syntax does, which means the only valid values are integers, so I think this option should be adjusted to reflect that (and make the correct conversion with `toString` in the exec script).